frontend/src/routes/(authed)/player/[id]/+page.server.ts

```diff
@@ -8,18 +9,45 @@ export const load: PageServerLoad = async ({
   if (Number.isNaN(playerId)) {
     throw new Error('Invalid player ID');
   }
-  const [matches, users] = await Promise.all([
+  const [matches, users, mmrHistory] = await Promise.all([
     apiClient.mmrApi.v2MmrMatchesGet({
-      userId: Number(playerId),
-      limit: 100,
+      userId: playerId,
+      limit: 1000,
       offset: 0,
     }),
     apiClient.usersApi.v1UsersGet(),
+    apiClient.statisticsApi.v1StatsPlayerHistoryGet({ userId: playerId }),
   ]);
 
+  const user = users.find((user) => user.userId === playerId);
+  if (!user) {
+    throw new Error('Player not found');
+  }
+
+  const totalMatches = matches.length;
```

Comment:
Isn't it better to add something to the stats controller that can return stats for a single user? For now it's not an issue because people have not played more than 1000 matches.

Reply:
Yes, that would be better, but let's do it later. I don't know how performance will be when we have +100 matches - maybe you can try?